### PR TITLE
Crossbuild for 2.13.0

### LINKED ISSUE #805: publish for scala 2.13.0
scala 2.13.0 released!  \o/

### PR DESCRIPTION
**Pull request by gabro**
Fixes #805

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -494,7 +494,10 @@ lazy val commonScalacOptions = Def.setting(
     case Some((2, v)) if v >= 13 =>
       Seq()
     case _ =>
-      Seq("-Yno-adapted-args")
+      Seq(
+        "-Yno-adapted-args",
+        "-Xfuture"
```
Comment (gabro) [inline build.sbt]: `-Xfuture` is deprecated in 2.13.x

**Comment by gabro** [comment]
I have another PR pending which replaces the unicode identifiers ←, ⇒ and →, which are now deprecated in Scala 2.13.

The net benefit is reducing the number of warnings when building against 2.13.

@larsrh can I push those changes here, and should I open a separate PR?

**Comment by larsrh** [comment]
> can I push those changes here, or should I open a separate PR?

Another PR would be great.

**Comment by gabro** [comment]
Sure 👍 I'll open it as soon as this one is in. (I've accidentally closed it and reopened it... fat thumb!)

**Comment by erikerlandson** [comment]
Thanks @gabro !  :tada:

**Pull request merged**